src/test/java/io/confluent/kafka/connect/datagen/DatagenTaskTest.java

```diff
@@ -373,10 +373,11 @@ private void loadKeyAndValueSchemas(String schemaResourceFilename, String idFiel
 
   private org.apache.avro.Schema loadAvroSchema(String schemaFilename) {
     try {
-      Generator generator = new Generator(
-          getClass().getClassLoader().getResourceAsStream(schemaFilename),
-          new Random()
-      );
+      InputStream schemaStream = getClass().getClassLoader().getResourceAsStream(schemaFilename);
+      Generator generator = new Generator.Builder()
+              .schemaStream(schemaStream)
+              .random(new Random())
+              .generation(0L).build();
       return generator.schema();
```

Comment:
I think I left this comment before when reviewing https://github.com/confluentinc/kafka-connect-datagen/pull/64; we don't need to construct a new `Generator` just to parse an Avro schema:
```suggestion
      InputStream schemaStream = getClass().getClassLoader().getResourceAsStream(schemaFilename);
      return org.apache.avro.Schema.Parser().parse(schemaStream);
```

Reply:
I didn't understand that either, but I was passing by quickly...